pipelines/skylab/build_indices/BuildIndices.changelog.md

```diff
@@ -1,3 +1,7 @@
+# 1.0.1
```

Comment:
Let's rollback this change so that we don't publish a new version of the pipeline that doesn't run